packages/js/eslint-config/package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.1.0",
   "description": "",
   "main": "index.js",
+  "bin": {
+    "eslint": "./node_modules/eslint/bin/eslint.js"
```

Comment:
Could you please add a note (maybe into the commit message) about why we need to export explicitly?

Reply:
Thank you for the explanation. I was guessing it has something to do with the premium.